versioned/transfer/src/main/java/org/projectnessie/versioned/transfer/ImportPersistV23.java

```diff
@@ -160,7 +160,7 @@ void processGeneric(RelatedObj genericObj) throws ObjTooLargeException {
 
     Obj obj;
     if (type.equals(StandardObjType.UNIQUE)) {
-      obj = UniqueIdObj.uniqueId(id, genericObj.getSpace(), genericObj.getData());
+      obj = UniqueIdObj.uniqueId(id, 0L, genericObj.getSpace(), genericObj.getData());
```

Comment:
Looks suspicious / wrong? Shouldn't we modify `RelatedObj` and include a `referenced` field?

Reply:
We're importing here. The `Persist` implementations store those with the current referenced timestamp.